### DISCUSSION TITLE
Display a link from the item to his data model

### DISCUSSION THREAD

**Discussion by ClemenF** [directus/directus#23823] [Feature Requests]
### Summary

Hi

While creating a whole new content model, we often go back and forth between editing and the data model. It would be convenient to have a link to the data model of the item currently being edited.

### Motivation

While I'm editing content, I sometimes realise that I'm missing a field or that I should modify one.
If I want to do it, i have to leave my content, go to the content model page, find my content model and select it. This could be a gain of time to have a link in the content that will redirect me to the content model editor.

### Detailed Design

I see 3 options : 
![image](https://github.com/user-attachments/assets/d5bc8b01-c3c3-4232-83f3-5acec3fb109e)

1. Change the link to go to the content model
2. Add a round button near the save one
3. Add a new button in the right tab

### Should Have:
- A option to be redirect to the content model

### Drawbacks

I think one of the drawbacks, is that the content editor could accidently click on the link and lost his content. Maybe, the link should be only display to administrator.

### Alternatives

I don't know.

### Adoption Strategy

It is not a breaking change. It's a UX help.

**Comment by rijkvanzanten** [comment]
![CleanShot 2024-12-03 at 18 43 07@2x](https://github.com/user-attachments/assets/fb4af20e-8d8f-4a45-82c9-042c6efc117e)

Directus does currently already have a right-click to edit 🙂 Adding a more prominent button is not a bad idea though 🤔
